press-sync.php

```diff
@@ -239,26 +239,28 @@ public function get_objects_to_sync( $objects_to_sync, $next_page = 1, $taxonomi
 	}
 
 	/**
-	 * Return the posts to sync
+	 * Return the posts to sync.
 	 *
 	 * @since 0.1.0
 	 *
-	 * @param string $objects_to_sync
-	 * @param integer $next_page
-	 * @param array $taxonomies
-	 * @return array posts
+	 * @param string  $objects_to_sync The objects to sync.
+	 * @param integer $next_page       The next page of results.
+	 * @param array   $taxonomies      The related taxonomies for the objects.
+	 * @param string  $where_clause    An SQL Where clause to modify the query to retrieve posts.
+	 *
+	 * @return array  $posts           The posts to return.
 	 */
 	public function get_posts_to_sync( $objects_to_sync, $next_page = 1, $taxonomies, $where_clause = '' ) {
 
 		global $wpdb;
 
 		$offset = ( $next_page > 1 ) ? ( $next_page - 1 ) * 5 : 0;
 
-		$where_clause 	= ( $where_clause ) ? ' AND ' . $where_clause : '';
-		$sql 			= "SELECT * FROM $wpdb->posts WHERE post_type = %s AND post_status NOT IN ('auto-draft','trash') $where_clause ORDER BY post_date DESC LIMIT 5 OFFSET %d";
-		$prepared_sql 	= $wpdb->prepare( $sql, $objects_to_sync, $offset );
+		$where_clause   = ( $where_clause ) ? ' AND ' . $where_clause : '';
+		$sql            = "SELECT * FROM {$wpdb->posts} WHERE post_type = %s AND post_status NOT IN ('auto-draft','trash') $where_clause ORDER BY post_date DESC LIMIT 5 OFFSET %d";
```

Comment:
Same note to wrap `$where_clause` - essentially any time you interpolate a string in PHP you should wrap it in `{}` for clarity and readability.